lambdas/data-migration2/src/pdrs.ts

```diff
@@ -69,39 +69,39 @@ export const migratePdrRecord = async (
 
   const providerCumulusId = await providerPgModel.getRecordCumulusId(
     knex,
-    { name: dynamoRecord.provider }
+    { name: dynamoPDR.provider }
   );
 
-  const executionCumulusId = dynamoRecord.execution
+  const executionCumulusId = dynamoPDR.execution
     ? await executionPgModel.getRecordCumulusId(
       knex,
-      { arn: dynamoRecord.execution }
+      { url: dynamoPDR.execution }
```

Comment:
This is the actual fix.

Reply:
The only other thought I have here is that the PDR migration will fail if there is a `dynamoPdr.execution` but we can't find a matching record. That is perfectly logical, but I notice in the granules migration we have updated the logic to not fail granules migration if the execution cannot be found

Since execution isn't a required field for PDRs, we could do the same thing here. It's not a bug and not something we have to change, just thinking out loud. Maybe it's easier to limit the scope here for now and leave it as is

Reply:
I could do this in a follow-up PR. It seems worthwhile, even if it's just to match behavior in other migrations. But yeah, it might be best to not hold this up.